AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt

```diff
@@ -130,12 +131,11 @@ private fun ExploreList(
             }
         }
         item {
```

Comment:
I think this would be better done as contentPadding on the LazyColumn and something like

`contentPadding = WindowInsets.navigationBars.asPaddingValues()`